src/index.html

```diff
@@ -21,7 +24,394 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__budy">
```

Comment:
There is a typo in the class name `page__budy`. It should likely be `page__body`.